plugin/slime.vim

```diff
@@ -9,9 +9,9 @@ let g:loaded_slime = 1
 
 command -bar -nargs=0 SlimeConfig call slime#config()
 command -range -bar -nargs=0 SlimeSend call slime#send_range(<line1>, <line2>)
-command -nargs=+ SlimeSend1 call slime#send(<q-args> . "\r")
+command -nargs=+ SlimeSend1 call slime#send(<q-args> . "\n")
 command -nargs=+ SlimeSend0 call slime#send(<args>)
-command! SlimeSendCurrentLine call slime#send(getline(".") . "\r")
+command! SlimeSendCurrentLine call slime#send(getline(".") . "\n")
```

Comment:
this would have consequences beyond the tmux plugin…

Reply:
If you have a suggestion to tailor it just for tmux I can improve the pull, I only ever use vim-slime with tmux and nothing else.

Reply:
hi @oblitum,

This is tricky. I try to keep an eye on what problems people report and how often these people happen. Then, I need to keep in mind "what could break" for any given change … and how many people would be affected. (it doesn't help that there's an explosion of configurations out there: which vim, which OS, which terminal, which REPL, etc.)

I can leave this PR open and see how many people it helps (or not).

In the meantime, I tried to fix a workaround for customizations without having people maintain their own fork: [overrides](https://github.com/jpalardy/vim-slime#advanced-configuration-overrides).

Reply:
Okay thanks. Actually this pull does the job for me already, can't be easier than what's already written right? I then can stick to my fork and occasionally rebase this commit on top of upstream... I don't have much knowledge on vim-slime code base, so I thought that adding back something that was on master before wouldn't be big deal.

Reply:
On a side note, there's some issues involving tmux open, given that bracketed paste changes how tmux is handled substantially, _maybe_ it can make vim-slime better (or worse) regarding those issues.

Reply:
Like #245, but it would require that reporters there give this a try.